vaadinfrontend/src/main/java/life/qbic/datamanager/views/support/experiment/ExperimentItem.java

```diff
@@ -0,0 +1,132 @@
+
+/**
+ * <b><class short description - 1 Line!></b>
+ *
+ * <p><More detailed description - When to use, what it solves, etc.></p>
+ *
+ * @since <version tag>
+ */
+public class ExperimentItem extends Card {
+
+  @Serial
+  private static final long serialVersionUID = 1787087141809506158L;
+
+  private final Experiment experiment;
+
+  private final Span activeTag = new Span("active");
+
+  private final Span experimentTitle = new Span("no title set");
+
+  private final Div progressSection = new Div();
+
+  private final Div contentSection = new Div();
+
+  private final Div activeSection = new Div();
+
+  private final Icon flaskIcon = VaadinIcon.FLASK.create();
+
+  private final Span progressTag = new Span("no progress set");
+
+  private final List<ComponentEventListener<ExperimentItemClickedEvent>> selectionListeners = new ArrayList<>();
+
+  private ExperimentItem(Experiment experiment) {
+    this.experiment = experiment;
+    layoutComponent();
+    configureComponent();
+  }
+
+  private void configureComponent() {
+    loadProgressStatus();
+    loadExperimentTitle();
+    setStatusToInactive();
+    addListener(ExperimentItemClickedEvent.class, listener -> informListeners(new ExperimentItemClickedEvent(this, true)));
+  }
+
+  private void setStatusToInactive() {
+    activeTag.setVisible(false);
+  }
+
+  private void loadProgressStatus() {
+    progressSection.addClassName("incomplete-experiment");
+    progressSection.setText("incomplete");
+  }
+
+  private void loadExperimentTitle() {
+    experimentTitle.setText(experiment.getName());
+  }
+
+  private void layoutComponent() {
+    addClassName("experiment-item");
+    layoutProgressStatus();
+    layoutExperimentLabel();
+    layoutActiveSection();
+  }
+
+  private void layoutActiveSection() {
+    activeTag.getElement().getThemeList().add("badge success primary");
+    activeTag.addClassName(BorderRadius.MEDIUM);
+    activeSection.add(activeTag);
+    activeSection.addClassName("active-section");
+    add(activeSection);
+  }
+
+  private void layoutExperimentLabel() {
+    experimentTitle.addClassName("experiment-title");
+    contentSection.addClassName("content-section");
+    contentSection.add(experimentTitle);
+    contentSection.add(flaskIcon);
+    add(contentSection);
+  }
+
+  private void layoutProgressStatus() {
+    progressSection.addClassName("progress-section");
+    progressTag.addClassName("progress-tag");
+    progressSection.add(progressTag);
+    add(progressSection);
+  }
+
+  public static ExperimentItem create(Experiment experiment) {
+    Objects.requireNonNull(experiment);
+    return new ExperimentItem(experiment);
+  }
+
+  public ExperimentId experimentId() {
+    return experiment.experimentId();
+  }
+
+  public void setAsActive() {
+    activeTag.setVisible(true);
+  }
+
+  public void setAsInactive() {
+    activeTag.setVisible(false);
+  }
+
+  public void setAsSelected() {
+    addClassNames("selected");
+  }
```

Comment:
I'd recommend a dedicated removeSelection() method to remove the set className. Currently it works because selecting an item triggers a reload of the page with all experimentItems, but this does not have to be the case in the future.

Reply:
Since no given scenario demands it, no need imo :) Can be added if required.